### DISCUSSION TITLE
多页面插件 umi-plugin-multitabs

### DISCUSSION THREAD

**Discussion by SagittariusZhu** [umijs/umi#7271] [Ideas]
写了一个无侵入的多标签页插件，方便有需要的同学使用。[github](https://github.com/SagittariusZhu/umi-plugin-multitabs)

## Install

Using npm:

```bash
$ npm install --save-dev umi-plugin-multitabs
```

or using yarn:

```bash
$ yarn add umi-plugin-multitabs --dev
```

## Usage

Modify `config/config.js` or `.umirc.js`

```
export default defineConfig({
  ...
  multitabs: {},
});
```

This plugin is enabled by config.

Now you have multitabs support.

## Dependencies

You must use umi3 and @ant-design/pro-layout.

**Comment by pxim** [comment]
要用这个插件的话，代码结构有什么要求么？
![image](https://user-images.githubusercontent.com/5111701/151491950-a1193f47-8b3f-41fa-8e5a-672178ac81ad.png)
